src/index.d.ts

```diff
@@ -0,0 +1,693 @@
+import { Script, TxOut } from 'bsv';
+
+declare interface CastTemplate {
+  lockingScript: {
+    script: unknown[];
+    size: ((params: object) => number) | number;
+    validate: (params: object) => void;
+  }
+}
+
+/**
+ * Forge transaction builder class.
+ */
+declare class Forge {
+  /**
+   * Instantiates a new Forge instance.
+   * 
+   * The accepted params are:
+   * 
+   * * `inputs` - list of input objects or cast instances
+   * * `outputs` - list of output objects or cast instances
+   * * `changeTo` - address to send change to
+   * * `changeScript` - bsv Script object to send change to
+   * * `options` - set `rates` or `debug` options
+   * 
+   * @param params Tx parameters
+   * @constructor
+   */
+  constructor(params?: Partial<{
+    inputs: object[] | Cast[];
+    outputs: object[] | Cast[];
+    changeTo: string;
+    changeScript: unknown;
+    nLockTime: number;
+    options: Partial<{
+      rates: unknown;
```

Comment:
The `rates` property is an object that looks like this:

```
{
  mine: {
      data: number,
      standard: number
    },
    relay: {
      data: number,
      standard: number
    }
}
```

Reply:
Sorted